### PR TITLE
[Fixtures] Country factory

### PR DESCRIPTION
**Pull request by loic425**
| Q               | A
| --------------- | -----
| Branch?         | foundry-fixtures
| Bug fix?        | no
| New feature?    | yes
| BC breaks?      | no
| Deprecations?   | no
| Related tickets | 
| License         | MIT

### CODE REVIEW

File: src/Sylius/Bundle/CoreBundle/DataFixtures/Factory/CountryFactory.php [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+                $country->setEnabled($attributes['enabled']);
+
+                return $country;
+            })
+        ;
+    }
+
+    protected static function getClass(): string
+    {
+        return Country::class;
```
Comment (vvasiloi) [inline src/Sylius/Bundle/CoreBundle/DataFixtures/Factory/CountryFactory.php:86]: I missed it in the previous PRs, but how does that work in the end-user application where all the resources classes are extended? Normally, parameters like `sylius.model.country.class` would be used.

File: tests/DataFixtures/Factory/CountryFactoryTest.php [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+    {
+        $country = CountryFactory::new()->create();
+
+        $this->assertInstanceOf(CountryInterface::class, $country->object());
+        $this->assertNotNull($country->getCode());
+        $this->assertTrue(Countries::exists($country->getCode()));
+    }
+
+    /** @test */
+    function it_creates_country_with_custom_code(): void
```
Comment (vvasiloi) [inline tests/DataFixtures/Factory/CountryFactoryTest.php:39]:
```suggestion
    function it_creates_country_with_given_code(): void
```
It's not really a custom code.

File: src/Sylius/Bundle/CoreBundle/DataFixtures/Factory/CountryFactory.php (hunk 1 above)
Comment (loic425) [reply src/Sylius/Bundle/CoreBundle/DataFixtures/Factory/CountryFactory.php:86]: Yes I know that, this `getClass` method is required. But the problem is, it is a static method, so we just can not use the "sylius.model.country.class" parameter. But with using the `instantiateWith` callable which uses the entity factory, I think this string does not change anything.

Comment (vvasiloi) [reply src/Sylius/Bundle/CoreBundle/DataFixtures/Factory/CountryFactory.php:86]: Alright! We can also think about this later if it's an issue.

Comment (loic425) [reply src/Sylius/Bundle/CoreBundle/DataFixtures/Factory/CountryFactory.php:86]: I added it on the main "issue".

**Pull request merged**